.github/workflows/build-artifacts-code.yml

```diff
@@ -62,5 +62,4 @@ jobs:
           path: |
             src/out/*
       - name: Publish Nuget packages to GitHub registry
-        working-directory: src
```

Comment:
It is possible this was causing the problem also -- if `setup-dotnet` added a `NuGet.config` file to the root, but the working directory was `src`, then it may not have been able to find the `NuGet.config` file with the password... just another guess of course.